tests/api_fastapi/core_api/routes/public/test_dag_parsing.py

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING
+
+import pytest
+from sqlalchemy import select
+
+from airflow.models import DagBag
+from airflow.models.dagbag import DagPriorityParsingRequest
+from airflow.utils.session import provide_session
+
+from tests_common.test_utils.db import clear_db_dag_parsing_requests
+
+pytestmark = [pytest.mark.db_test, pytest.mark.skip_if_database_isolation_mode]
+
+if TYPE_CHECKING:
+    from airflow.models.dag import DAG
+
+
+class TestDagParsingRequest:
```

Comment:
Just to be consistent with other test classes.
```suggestion
class TestDagParsingEndpoint:
```

Reply:
Changed the class name